facebook/OpenBIC
# wc: mb: Support gpio

Summary:
- GPIO function enable
- Support DC status on/off.

Dependency: https://github.com/facebook/OpenBIC/pull/386

Test plan:
- Build code: PASS
- Check system DC flag will change: Pass

Log:
- power on
  ```
  BMC:
  root@bmc-oob:~# power-util mb on
  Powering fru 1 to ON state...
  root@bmc-oob:~# power-util slot1 status
  Power status for fru 1 : ON
  root@bmc-oob:~# bic-util mb 0xe0 0x41 0x15 0xa0 0x00 0x00 0x49
  15 A0 00 A3 01
  
  BIC:
  uart:~$ [set_DC_status] gpio number(39) status(1)
  [set_post_status] gpio number(1) status(1)
  ```
- power off
  ```
  BMC:
  root@bmc-oob:~# power-util slot1 off
  Powering fru 1 to OFF state...
  root@bmc-oob:~# power-util slot1 status
  Power status for fru 1 : OFF
  root@bmc-oob:~# bic-util mb 0xe0 0x41 0x15 0xa0 0x00 0x00 0x49
  15 A0 00 A3 00
  BIC:
  uart:~$ [set_post_status] gpio number(1) status(0)
  [set_DC_status] gpio number(39) status(0)
  ```
